starter-code/aboutMe.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sultan Alsubaie | About Me</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <hr id="big">
+       <div class="headerA"> 
+           <a href="aboutMe.html">About Me</a> |
+         <a href="portfolio.html">Portfolio</a>
+        </div>
+        <h1>SULTAN ALSUBAIE</h1>
+
+    </header>
+    <main class="clearfix">
+        <hr>
+        <div class="text">
+            <h2>I ENJOY LIFE AS A DEVELOPER</h2>
+            <p>I'm Sultan Alsubaie, a Computer Information Systems major, but a passionate Developer
+                based in Khobar. I have 4 years of expereince in the graphic design world, and specializing
+                in the creation of responsive websites.
+            </p>
+        </div>
+        <div class="image"><img src="images/personalImage.png" alt="personalPicture"></div>

```

Comment:
Mr. griffin your personalImage ends with `.jpg` but you add `.png` , also you didn't style it right